src/internt-vedtak/typer/vilkårperiode.ts

```diff
@@ -80,7 +80,7 @@ export const resultatDelvilkårperiodeTilTekst: Record<ResultatDelvilkårperiode
     IKKE_AKTUELT: 'Ikke aktuelt',
 };
 
-enum TypeStønadsperiode {
+export enum TypeStønadsperiode {
```

Comment:
Burde TypeStønadsperiode renames til TypeVilkårsperiode?

Reply:
Ja, det kan renames. Foreløbig brukes den til både vedtaksperioder og stønadsperioder (enn så lenge læremidler bruker stønadsperioder). Jeg kommer ikke på noe bra felles ord for vedtaksperioder og stønadsperioder, har du noen forslag?

Hvis ikke foreslår jeg at vi lar det stå til vi har migrert læremidler også. Da kan vi rename den til `TypeVedtaksperiode` og fjerne stønadsperiode helt fra koden